webargs/fields.py

```diff
@@ -36,3 +36,19 @@ def __init__(self, nested, *args, **kwargs):
         if isinstance(nested, dict):
             nested = argmap2schema(nested)
         super(Nested, self).__init__(nested, *args, **kwargs)
+
+class DelimitedList(ma.fields.List):
+
+    delimiter = ','
+
+    def __init__(self, cls_or_instance, delimiter=None, **kwargs):
+        self.delimiter = delimiter or self.delimiter
+        super(DelimitedList, self).__init__(cls_or_instance, **kwargs)
+
+    def _serialize(self, value, attr, obj):
+        ret = super(DelimitedList, self)._serialize(value, attr, obj)
+        return self.delimiter.join(ret)
```

Comment:
I'm not sure this should serialize to a string by default, In the typical use case--rendering an HTTP response with serialized data--wouldn't you usually want a list?

In any case, I think a parameter should be added to modify this behavior, e.g `as_string`.